packages/jest-serializer-filepath/src/index.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+const cwd = process.cwd();
```

Comment:
This is pretty cool but I don't think using `process.cwd()` is sufficient. The directory can change based on where Jest is invoked which would lead to outdated snapshots all the time. I'm wondering if we could make it configurable somehow?